### PR TITLE
Update measure model

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-validators require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: .simplecov [hunk 1]
```diff
@@ -6,5 +6,5 @@ if ENV['CI'] == 'true'
   require 'codecov'
   SimpleCov.formatter = SimpleCov::Formatter::Codecov
 else
-  SimpleCov.formatter = SimpleCov::Formatter::QualityFormatter
+  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter
```
Comment (okeefm) [inline .simplecov]: This change throws an error at the end of my test suite: `simplecov-0.16.1/lib/simplecov/result.rb:48:in `format!': private method `format' called for #<Class:0x00007fa1cd8c5ae8> (NoMethodError)`. We may want this to be `HTMLFormatter` instead.

File: lib/data_validator.rb [hunk 2]
```diff
@@ -49,32 +49,27 @@ class DataValidator
...
     end
 
     def validate(file, options = {})
       doc = get_document(file)
 
       doc.xpath('//*[@sdtc:valueSet]').each_with_object([]) do |node, errors|
         oid = node.at_xpath('@sdtc:valueSet')
-        vs = @bundle.value_sets.where('oid' => oid).first
+        vs = CQM::ValueSet.where('oid' => oid, '_id' => { '$in' => @value_set_ids }).first
+        next unless vs
```
Comment (okeefm) [inline lib/data_validator.rb]: It looks like not finding the valueset used to be an error. Should this still be an error?

File: lib/data_validator.rb [hunk 3]
```diff
@@ -49,32 +49,27 @@ class DataValidator
...
+
         code = node.at_xpath('@code')
-        code_system = node.at_xpath('@codeSystem')
+        code_system_oid = node.at_xpath('@codeSystem')
         null_flavor = node.at_xpath('@nullFlavor')
-        if !vs
-          errors << build_error("The valueset #{oid} declared in the document cannot be found", node.path, options[:file_name])
-        elsif !@oids.include?(oid.value)
-          errors << build_error("File appears to contain data criteria outside that required by the measures. Valuesets in file not in measures tested #{oid}'",
+        next unless vs.concepts.where(code: code, code_system_oid: code_system_oid).count.zero?
```
Comment (okeefm) [inline lib/data_validator.rb]: Should this be an error, rather than a `next` as well? I'm assuming this is the equivalent of the removed `@oids.include?(oid.value)` check

File: lib/qrda_qdm_template_validator.rb [hunk 4]
```diff
@@ -4,207 +4,7 @@ module CqmValidators
   class QrdaQdmTemplateValidator
     include BaseValidator
 
-    # Hash of templateIds/extensions specified in the Patient Data Section QDM for QRDA R3
```
Comment (okeefm) [inline lib/qrda_qdm_template_validator.rb]: Just checking that we don't anticipate QRDA Cat I <R5 being used anymore? Either for our projects, or any other users of this library?

File: test/unit/measure_validator_test.rb [hunk 5]
```diff
@@ -53,12 +69,6 @@ class MeasureValidatorTest < ActiveSupport::TestCase
     assert_equal 2, errors.length
   end
 
-  test 'should have no errors if set id is valid (cat 1)' do
```
Comment (okeefm) [inline test/unit/measure_validator_test.rb]: What was the reasoning behind removing this test? (Not saying it shouldn't be removed, just curious to know why)

Comment (dczulada) [reply test/unit/measure_validator_test.rb]: It was unclear to me what differentiated this test from the 'should have no errors if measure id is valid (cat 1)' test.

Comment (dczulada) [reply test/unit/measure_validator_test.rb]: It was unclear to me what differentiated this test from the 'should have no errors if measure id is valid (cat 1)' test.

File: lib/qrda_qdm_template_validator.rb (hunk 4 above)
Comment (dczulada) [reply lib/qrda_qdm_template_validator.rb]: Doing this to be consistent with the libraries.  For older versions of QRDA, they would use an older version of this library.

File: lib/data_validator.rb (hunk 2 above)
Comment (dczulada) [reply lib/data_validator.rb]: This error was then later ignored by Cypress.  Gone are the days of a restrictive 'Scoop and Filter' where one was required to filter out information not related to a measure.

File: lib/data_validator.rb (hunk 3 above)
Comment (dczulada) [reply lib/data_validator.rb]: The "next" here is triggered when a code/codeSystem is found in a ValueSet.
